packages/manager/modules/log-to-customer/src/list/template.html

```diff
@@ -0,0 +1,154 @@
+<oui-header
+    heading="{{:: $ctrl.title }}"
+    description="{{ :: $ctrl.description }}"
+>
+    <oui-back-button data-ng-click="$ctrl.$state.go('^')"> </oui-back-button>
```

Comment:
don't use `$state.go` directly in the template create a function in the controller instead